xml/System.Web.UI.WebControls/LinqDataSourceView.xml

```diff
@@ -1842,9 +1961,17 @@
         <ReturnType>System.Web.UI.WebControls.ParameterCollection</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets a collection of parameters that are used to create the OrderBy clause.</summary>
```

Comment:
I think "OrderBy" needs code brackets to prevent translation. This same problem is in the `<value>` node below.

Reply:
Ditto